cmd/kube-proxy/app/server_linux.go

```diff
@@ -122,11 +124,17 @@ func (s *ProxyServer) platformCheckSupported(ctx context.Context) (ipv4Supported
 			logger.Info("No iptables support for family", "ipFamily", v1.IPv6Protocol)
 		}
 	} else {
-		// Assume support for both families.
-		// FIXME: figure out how to check for kernel IPv6 support using nft
+		// The nft CLI always supports both families.
 		ipv4Supported, ipv6Supported = true, true
 	}
 
+	// Check if the OS has IPv6 enabled, by verifying if the IPv6 interfaces are available
+	_, errIPv6 := os.Stat("/proc/net/if_inet6")
```

Comment:
I see, this seems also adopted for other similar cases https://github.com/vmware/photon/commit/1a85b73b5107db5cd3bb83880dda32242e122884

/lgtm

/assign @danwinship 
for approval